02-microarray/pathway-analysis_microarray_03_gsva.Rmd

```diff
@@ -376,20 +394,21 @@ max_dup_df <- mapped_df %>%
 max_dup_df
 ```
 
-We can see `GSM187153` now has the `-0.007355525` value for Entrez ID `6013` like expected.
-
+We can see `GSM917111` now has the `0.2294387` value for Entrez ID `6013` like expected. 
 Looks like we were able to successfully get rid of the duplicate Entrez gene identifiers!
 
 Now let's combine our de-duplicated data with the rest of the mapped data!
 
 ```{r}
 filtered_mapped_df <- mapped_df %>%
+  # We won't be using Ensembl IDs moving forward, so we will drop this column
+  dplyr::select(-Ensembl) %>%
```

Comment:
You will see, I brought this back. Because we show this in a preview I don't want users think that this whichever Ensembl ID actually makes it into the data frame has anything to do with anything. 

I realize this means we drop it twice, but I think it makes the data frame less confusing for the preview.